x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -358,26 +354,26 @@ export function AlertDetails() {
       'data-test-subj': 'relatedAlertsTab',
       content: <RelatedAlerts alertData={alertDetail} />,
     },
-    ...(validDashboards?.length
-      ? [
-          {
-            id: RELATED_DASHBOARDS_TAB_ID,
-            name: (
-              <>
-                <FormattedMessage
-                  id="xpack.observability.alertDetails.tab.relatedDashboardsLabel"
-                  defaultMessage="Related dashboards"
-                />{' '}
-                <EuiNotificationBadge color="success">
-                  {validDashboards?.length}
-                </EuiNotificationBadge>
-              </>
-            ),
-            'data-test-subj': 'relatedDashboardsTab',
-            content: relatedDashboardsTab,
-          },
-        ]
-      : []),
+    {
```

Comment:
Before we were not showing the tab if there were no linked dashboards. With this change, I understand we will always show the tab, independent if we have related dashboards or not. Do we handle the empty state and what do we display in this case? I am half way through the review, so I might get the answer!